server/test/services/migration/ProgramMigrationServiceTest.java

```diff
@@ -171,8 +171,11 @@ public void maybeGenerateNewAdminName_generatesCorrectAdminNames() {
     resourceCreator.insertQuestion("name-question-1");
     resourceCreator.insertQuestion("name-question-2");
 
-    String newAdminName = service.maybeGenerateNewAdminName("name-question");
-    assertThat(newAdminName).isEqualTo("name-question-3");
+    String newAdminNameOne = service.maybeGenerateNewAdminName("name-question");
+    assertThat(newAdminNameOne).isEqualTo("name-question-3");
+
+    String newAdminNameTwo = service.maybeGenerateNewAdminName("name-question-1");
+    assertThat(newAdminNameTwo).isEqualTo("name-question-3");
```

Comment:
**suggestion:** Would you mind separating this second test case into it's own test? I find it a little confusing that newAdminNameOne and newAdminNameTwo end up resolving to the same name, but it makes sense cause its just calling the function not actually saving anything. But I think it would be clearer having one `maybeGenerateNewAdminName_generatesCorrectAdminNames_withoutNumberSuffix` and the other `maybeGenerateNewAdminName_generatesCorrectAdminNames_withNumberSuffix` or something.

Reply:
done!